assemblies/platform-plugin/src/main/resources/resources/web/common-ui-require-js-cfg.js

```diff
@@ -34,6 +34,7 @@
   var requirePaths = requireCfg.paths;
   var requireShim = requireCfg.shim;
   var requireMap = requireCfg.map;
+  var requirePackages = requireCfg.packages;
```

Comment:
https://github.com/dcleao/pentaho-platform-plugin-common-ui/blob/c0b99a9000438678e38d225792255d5d5765142f/assemblies/platform-plugin/src/main/resources/resources/web/common-ui-require-js-cfg.js#L189-L190

This mapping is no longer needed after this story